src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporterHelperExtensions.cs

```diff
@@ -59,49 +59,46 @@ public static OpenTelemetryLoggerOptions AddOtlpExporter(this OpenTelemetryLogge
 
         configureExporterAndProcessor?.Invoke(exporterOptions, processorOptions);
 
-        return AddOtlpLogExporterInternal(
-             loggerOptions,
-             exporterOptions: exporterOptions,
-             processorOptions: processorOptions);
+        return loggerOptions.AddProcessor(BuildOtlpLogExporter(exporterOptions, processorOptions));
     }
 
-    private static OpenTelemetryLoggerOptions AddOtlpExporterInternal(
+    internal static OpenTelemetryLoggerOptions AddOtlpExporterInternal(
```

Comment:
Is this called by anything? Just looking at the diff it seems like it could be removed but I might have missed something.

Reply:
it is called from here
https://github.com/open-telemetry/opentelemetry-dotnet/blob/12c2436b6c99f07fb9abdf5583bed5d39de726d9/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporterHelperExtensions.cs#L45

Reply:
Gotcha. Can we leave it `private` in that case? If the external stuff is now calling `BuildOtlpLogExporter` that should be fine. Just a nit, not a blocker 😄

Reply:
Done